base/genimport/importer.go

```diff
@@ -205,25 +211,25 @@ func createImportFile(o *Output, pkgpath string, pkg *types.Package, mode Import
 		return ""
 	}
 
-	err := ioutil.WriteFile(filepath, buf.Bytes(), os.FileMode(0644))
```

Comment:
(these shouldn't strictly be necessary -- the local var will win -- although of course it may make things clearer.)

Reply:
I suspect the line number got screwed up on your comment. Is this in reference the rename `filepath` -> `f`? In which case, I agree, it's not necessary, it's just a best practice not to have the names collide. Doing the rename does up the chance of a merge conflict though, so I'm more than glad to undo it. We could also alias the pkg, or move the uses of it to the `github.com/cosmos72/gomacro/base/paths` pkg, which we already import and already uses `filepath`.

Reply:
Yep that's what I meant.  I think any option is fine, just wanted to mention it.  Another option is to move the modules code to a separate file, which might be good anyway, although I haven't thought a lot about it.